package/MDAnalysis/analysis/gnm.py

```diff
@@ -217,17 +219,25 @@ class GNMAnalysis(object):
           `Bonus_groups` is contained in `selection` as this could lead to
           double counting. No checks are applied. Default is ``None``.
 
+    Attributes
+    ----------
+    results : list
+        GNM results per frame:
+            results = [(time,eigenvalues[1],eigenvectors[1]),
+                       (time,eigenvalues[1],eigenvectors[1]), ...]
+
     See Also
     --------
     :class:`closeContactGNMAnalysis`
 
```

Comment:
```suggestion


```

That should be cause of the doc issue, the two empty lines are needed here to tell sphinx you are exiting from the `see also` block.